examples/fastify-ssr/src/server/graphql/resolvers.js

```diff
@@ -0,0 +1,25 @@
+const users = [
+  {
+    name: 'Brian'
+  },
+  {
+    name: 'Jack'
```

Comment:
@Joezo you should add your name to this 😆

Reply:
custom all-contributors integration? :trollface: